src/BlobField/blobs.js

```diff
@@ -0,0 +1,210 @@
+import paper from 'paper';
+
+import Ball from './ball';
+
+export default class Blobs {
+	constructor(artists, { onArtistHovered, onArtistClicked }) {
+		this.artists = artists;
+		this.balls = [];
+		this.numBalls = artists.length;
+		this.viewRatio = 0.75; // Ball will occupy viewRatio * viewArea of window
+		this.squeezeFactor = 0.3; // How squeeshy balls can be
+		this.animationDuration = 250;
+		this.mouseBall = null;
+		this.mouseCurrX = 0;
+		this.mouseCurrY = 0;
+		this.mouseTargetX = 0;
+		this.mouseTargetY = 0;
+		this.mouseRadiusMultiplier = 1.5; // Hidden mouse ball radius is * by this value
+		this.defaultEasing = 0.03; // Cursor position easing
+		this.slowSimEasing = this.defaultEasing / 30; // Cursor position easing on hover
+		this.opacity = 0.8;
+		this.hoverFadedOpacity = 0.2;
+		this.hoverActiveOpacity = 1;
+		this.isSlowSim = false;
+    this.collapsed = false;
+    this.onArtistHovered = onArtistHovered;
```

Comment:
changed from `setBGColor` for better separation of concerns

Reply:
Makes sense